e2e/cypress/e2e/tasks/tasksNotifications.cy.ts

```diff
@@ -87,10 +96,16 @@ describe('tasks notifications', () => {
 
     assertMessage('Task updated sucessfully');
 
-    getAssignedEmailNotification().then(({ toAddress, myTasksLink }) => {
-      assert(toAddress === 'organization.member@test.com', 'correct recipient');
-      cy.visit(myTasksLink);
-    });
+    getAssignedEmailNotification().then(
```

Comment:
I don't think e-mails should be tested primarily using e2e tests. It's a backend thing. 

You can keep this test, but in the future I would much rather see the stuff tested on the backend.

Reply:
The test was already there, I just added testing also the correct link to the task.

I agree that email testing should be done on BE, but at least one E2E is beneficial imho because it tests also the SMTP layer.